src/ui/public/system_api/system_api.js

```diff
@@ -0,0 +1,30 @@
+const SYSTEM_API_HEADER_NAME = 'kbn-system-api';
+
+export default function SystemApisProvider() {
```

Comment:
There's no reason to angularize this since it's just a native JS module, and it doesn't contain any state.  Instead, just export the two functions natively.